### ISSUE TITLE
Error in regex in validateUrl() on PHP 7.3.6

### ISSUE THREAD

**Issue by freescout-helpdesk** [laravel/framework#28900]
- Laravel Version: 5.5
- PHP Version: 7.3.6
- Database Driver & Version: Does not matter

### Description:

Here https://github.com/freescout-helpdesk/freescout/issues/31#issuecomment-503825103 we have the following error in PHP 7.3.6:
```
[2019-06-20 11:03:10] production.ERROR: preg_match(): Compilation failed: invalid range in character class at offset 1828 {"exception":"[object] (ErrorException(code: 0): preg_match(): Compilation failed: invalid range in character class at offset 1828 at /var/www/mailserver/vendor/laravel/framework/src/Illuminate/Validation/Concerns/ValidatesAttributes.php:1375)
```
Error is caused by:
```
return preg_match($pattern, $value) > 0;
```

Is it a known issue on 7.3.6?

### Steps To Reproduce:
Install PHP 7.3.6 and execure `validateUrl($attribute, $value)` function.

**Comment by staudenmeir** [comment]
This has been fixed in Laravel 5.5.43: #25197

**Comment by lksnmnn** [comment]
I have this exact same issue running Laravel 5.6.39 and PHP 7.3.6. The fix was merged in 5.6.34 and it the hyphen is escaped so there should be no regression there?!

The offset is still 1828 but the error is in line 1539 (ValidatesAttributes.php:1539)

```
reg_match(): Compilation failed: invalid range in character class at offset 1828","code":0,"file":"/var/www/app/releases/20190430090529Z/vendor/laravel/framework/src/Illuminate/Validation/Concerns/ValidatesAttributes.php:1539
```
